datasets/1.0/huggingface-squad/metadata.json

```diff
@@ -106,6 +113,11 @@
           "name": "record_set_plain_text/split",
           "description": "Split to which the example belongs to.",
           "dataType": "sc:Text",
+          "references": {
```

Comment:
nit: `references` coming after `source` was more readable.

Reply:
Yeah I agree... I don't know why it got changed, maybe it started to order those alphabetically? I could change it back manually, shall I do this?

Reply:
An array of strings used to control the order. We shouldn't do it manually but programmatically. But this is not prio at all - so we can probably let it like this if we don't have time.

Reply:
Resolved.